site/user-guide.md

```diff
@@ -66,6 +66,21 @@ You can have one or more operators in your Kubernetes cluster that manage one or
 We provide a Helm chart to manage the installation and configuration of the operator.
 Detailed instructions are available [here](install.md).
 
+## Exposing applications outside the Kubernetes cluster
+The operator can configure services to expose WebLogic applications and features outside of the Kubernetes cluster.  Care should be taken when exposing anything externally to ensure that the appropriate security considerations are taken into account. In this regard, there is no significant difference between a WebLogic domain running in a Kubernetes cluster and a domain running in a traditional data center.  The same kinds of considerations should be taken into account, for example:
+
+* Only expose those protocols and ports that need to be exposed.
+*	Use secure protocols (HTTPS, T3S, and such).
+*	Use custom channels to restrict the protocols that are exposed.
+*	Is load balancing required?
+*	Is certificate-based integrity needed?
+*	How will users authenticate?
```

Comment:
Add:
Is exposing RMI required (T3 enables RMI), or IIOP, or would HTTP alone suffice?
Is it necessary to expose admin capable channels?

Reply:
Is it required to expose default channels (these accept T3/RMI, and by default, accept admin traffic)?

Reply:
I have added your suggestions.